addons/portal/static/src/xml/portal_chatter.xml

```diff
@@ -81,11 +81,11 @@
             <t t-foreach="widget.get('messages') || []" t-as="message">
                 <div class="d-flex o_portal_chatter_message" t-att-id="'message-' + message.id">
                     <img class="o_portal_chatter_avatar" t-att-src="message.author_avatar_url" alt="avatar"/>
-                    <div class="flex-grow-1">
+                    <div class="flex-grow-1 w-75">
```

Comment:
Why is it needed?

Reply:
Hello @xavierbol 

If I don't define width then the name will pass through the boundary